test-data/unit/check-flags.test

```diff
@@ -509,3 +509,72 @@ x, y = 1, 2  # type: Unchecked, Unchecked
 [out]
 main:4: error: Type of variable becomes "Any" due to an unfollowed import
 main:6: error: A type on this line becomes "Any" due to an unfollowed import
+
+[case testDisallowAnyExprUnannotatedFunction]
+# flags: --disallow-any=expr
+def g(s):
+    return s  # E: Expressions of type "Any" are disallowed (has type "Any")
+
+g(0)
```

Comment:
After what we discussed [here](https://github.com/python/mypy/pull/3519/files/6517ae4055199642a7d3f8b52f81e7b37458995e#r121241083), it will be an error.